controllers/AdminController.php

```diff
@@ -59,7 +59,7 @@ public function actionCommonDrugs()
 	 * @param $ssd_id
 	 * @return html
 	 */
-	public function actionCommonDrugsDelete($ssdId)
+	public function actionCommonDrugsDelete($itemId)
```

Comment:
actually the listAutocomplete.js sending itemId, and the old version of the list need to be removed